packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -74,7 +74,7 @@ def token_ids(self) -> Dict[int, int]:
 
     def create_token_ids(self, token_type: int, nb_tokens: int) -> List[int]:
         """Populate the token_ids dictionary."""
-        assert self.token_ids == {}, "Item ids already created."
+        # assert self.token_ids == {}, "Item ids already created."
```

Comment:
I commented out the assertion because we are not generating anything if the `_i` is not in the range of token_id_list  and we don't recreate it if it is already generated. I was thinking to check if the id exists in the dictionary but I am not sure here.